apps/experiments/models.py

```diff
@@ -283,6 +289,22 @@ class Experiment(BaseTeamModel):
     """
 
     objects = ExperimentObjectManager()
+    status = models.CharField(
+        max_length=10,
+        choices=ExperimentStatus.choices,
+        default=ExperimentStatus.DRAFT,
+    )
+    working_experiment = models.ForeignKey(
+        "self",
+        on_delete=models.CASCADE,
+        null=True,
+        blank=True,
+        related_name="experiment_versions",
+    )
+    version_number = models.PositiveIntegerField(
+        null=True,
+        blank=True,
+    )
```

Comment:
note for future: it will be useful to be able to tell if edits have been made since the last 'version'. I see two options:

* A boolean: `has_changed` which get's set to `True` on every change
* An int: `patch_version` which get's incremented on every save